test/spec/modules/users/getMetadataByPublicAddress.spec.ts

```diff
@@ -11,3 +12,15 @@ test('#01: Successfully GETs to metadata endpoint via public address', async ()
   const getMetadataArguments = getMetadataStub.mock.calls[0];
   expect(getMetadataArguments).toEqual(['did:ethr:0x1234']);
 });
+
+test('#02: Successfully GETs to metadata endpoint via public address and wallet type', async () => {
```

Comment:
While the pattern here of `#2` for test cases exists today it is a pretty bad standard. This is fine for now but test names should stand alone without the numbering. 


Could we add negative tests? What happens if I pass in a nonexistent chain? or just something totally bad.

Reply:
Got it, I can remove the numbers from all the tests.

Each of the new functions require an argument of type WalletType so clients can only pass in chains we support. I have tests for SOLANA, and ANY but I can add one for NONE. There are negative tests for bad / unexpected inputs in DID parsing logic which we are reusing.